### ISSUE TITLE
[on hold] Normalize history items: 'See also's

### ISSUE THREAD

**Issue by jonkerz** [calacademy-research/antcat#842]
On hold, see https://antcat.org/issues/29#comment-85

Meta-issue: #779

- [x] After #916

#### type: `see_also`

Good candidate because it appears appears last in the list, and no `tax` tags.

Looks like this:

> See also: Brown & Kempf, 1969: 95; Bolton, 2000: 16

**Comment by jonkerz** [comment]
Closing - we do not want to have any 'See also' items.
